mappings/net/minecraft/block/AbstractBlock.mapping

```diff
@@ -82,6 +82,11 @@ CLASS net/minecraft/class_4970 net/minecraft/block/AbstractBlock
 		ARG 3 pos
 		ARG 4 context
 	METHOD method_9534 onUse (Lnet/minecraft/class_2680;Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_1657;Lnet/minecraft/class_1268;Lnet/minecraft/class_3965;)Lnet/minecraft/class_1269;
+		COMMENT Called whenever a player right-clicks (by default) a block. Is called once on the server and once on the client.
+		COMMENT <p>
+		COMMENT     When FAIL or PASS is returned on the client and the player is not holding an item , {@link net.minecraft.block.Block#onUse(net.minecraft.block.BlockState, net.minecraft.world.World, net.minecraft.util.math.BlockPos, net.minecraft.entity.player.PlayerEntity, net.minecraft.util.Hand, net.minecraft.util.hit.BlockHitResult)}  will be called twice on each side. It is best to return the same value on both the client and the server to prevent desyncs/rubber-banding. 
+		COMMENT </p>
+		COMMENT @return  {@link net.minecraft.util.ActionResult#SUCCESS} to make the player swing their hand and not use their equipped item. {@link net.minecraft.util.ActionResult#CONSUME} to make the player not swing their hand but still not use their item. {@link net.minecraft.util.ActionResult#FAIL} or {@link net.minecraft.util.ActionResult#PASS} to not swing the hand and use the equipped item. 
```

Comment:
```suggestion
		COMMENT Called whenever a player right-clicks (by default) a block. Is called once on the server and once on the client.
		COMMENT
		COMMENT <p>When FAIL or PASS is returned on the client and the player is not holding an item, {@link net.minecraft.block.Block#onUse(net.minecraft.block.BlockState, net.minecraft.world.World, net.minecraft.util.math.BlockPos, net.minecraft.entity.player.PlayerEntity, net.minecraft.util.Hand, net.minecraft.util.hit.BlockHitResult)} will be called twice on each side. It is suggested to return the same value on both the client and the server to prevent desyncs/rubber-banding.
		COMMENT
		COMMENT @return {@link net.minecraft.util.ActionResult#SUCCESS} to make the player swing their hand and not use their equipped item. {@link net.minecraft.util.ActionResult#CONSUME} to make the player not swing their hand but still not use their item. {@link net.minecraft.util.ActionResult#FAIL} or {@link net.minecraft.util.ActionResult#PASS} to not swing the hand and use the equipped item. 
```
Suggestions for spacing,